stock_orderpoint_uom/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Orderpoint UoM",
+    "summary": "Allows to create procurement orders in the UoM indicated in "
+               "the orderpoint",
+    "version": "11.0.1.0.0",
```

Comment:
@kittiu Change to '12.0.1.0.0'. This can relaunch tests correctly

Reply:
@rousseldenis thank you and fixed.